doc/src/records/dhwsys.md

```diff
@@ -221,12 +397,100 @@ Hot water delivery temperature (at output of water heater(s) and at point of use
 
 **wsTSetPointLH=*float***
 
-  Specifies the hot water setpoint temperature for all child DHWLOOPHEATERs.  Used only for HPWH-based DHWHLOOPEATERs (HPWH explicitly models tank temperatures and heating controls), otherwise has no effect.
+  Specifies the hot water set point temperature for all child DHWLOOPHEATERs.  Used only for HPWH-based DHWHLOOPEATERs (HPWH explicitly models tank temperatures and heating controls), otherwise has no effect.
 
   **Units**   **Legal Range**   **Default**   **Required**   **Variability**
   ----------- ----------------- ------------- -------------- -----------------
     ^o^F        $>$ 32 ^o^F      wsTSetPoint        No             hourly
 
+**wsTSetpointDes=*float***
+
+Specifies the design (sizing) set point temperature.
+
+<%= member_table(
+  units: "^o^F",
+  legal_range: "x $>$ 32 ^o^F",
+  default: "wsTUse",
+  required: "No",
+  variability: "constant") %>
+
+**wsVolRunningDes=*float***
+
+Running volume for design. Active volume (above aquastat) equals to a full tank volume, defaults from EcoSizer at end of prerun if not input. No direct use, must be passed to DHWHEATER via ALTER.
+
+<%= member_table(
+  units: "gal",
+  legal_range: "x $>$ 0",
+  default: "0.0",
+  required: "No",
+  variability: "constant") %>
+
+**wsASHPTSrcDes=*float***
+
+Design (sizing) source air temperature for HPWH.
+
+<%= member_table(
+  units: "^o^F",
+  legal_range: "x $>$ 32 ^o^F",
+  default: "Heating design temperature",
+  required: "No",
+  variability: "At the start and at the end of interval") %>
+
+**wsFxDes=*float***
+
+Excess size factor for domestic hot water design. wsFxDes is applied when wsHeatingCapDes and/or wsVolRunningDes are defaulted from EcoSizer at the end of the prerun. There is no effect if those values are input.
+
+<%= member_table(
+  units: "",
+  legal_range: "x $>$ 0",
+  default: "1.0",
+  required: "No",
+  variability: "constant") %>
+
+**wsDRMethod=*choice***
+
+Response control method for domestic hot water demand.
```

Comment:
Demand response control method.

See this [description of Demand Response](https://en.wikipedia.org/wiki/Demand_response#:~:text=Demand%20response%20is%20a%20reduction,peak%20and%20occasional%20demand%20spikes.).